chaincode/chaincode_ops/core/chaincode_ops.go

```diff
@@ -544,6 +564,61 @@ func (s *SmartContract) GetAllProposals(ctx contractapi.TransactionContextInterf
 	return proposals, nil
 }
 
+func (s *SmartContract) SetMaxMaliciousVotings(ctx contractapi.TransactionContextInterface, number int) error {
```

Comment:
The word "votings" looks odd. How about something like "SetMaxMaliciousOrgsInVotes"?